File: JAVA-Basic/JavaBasic/src/WhileLoop/Exercise/OldBooks.java
package WhileLoop.Exercise;

import java.util.Scanner;

public class OldBooks {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String searchingBook = scan.nextLine();
        String currentBook = scan.nextLine();
        int bookCounter = 0;

        while (!currentBook.equals(searchingBook) && !currentBook.equals("No More Books")) {
            bookCounter++;

            currentBook = scan.nextLine();
        }
        if (currentBook.equals("No More Books")) {
            System.out.println("The book you search is not here!");
            System.out.printf("You checked %d books.", bookCounter);
        } else {
            System.out.printf("You checked %d books and found it.", bookCounter);
        }
    }
}

File: 01.JS-Basic/WHILE-ЦИКЪЛ Упражнение/OldBooks.js
function oldBooks(input) {

    let book = String(input[0]);
    let checkedBook = String(input[1])
    let index = 2;
    let bookNum = 0;

    while (book !== checkedBook && checkedBook !== "No More Books") {
        checkedBook = input[index];
        bookNum++
        index++
    }
    if (checkedBook === "No More Books") {
        console.log("The book you search is not here!");
        console.log(`You checked ${bookNum} books.`)
    } else {
        console.log(`You checked ${bookNum} books and found it.`)
    }

}
oldBooks(["Troy",
"Stronger",
"Life Style",
"Troy"])


